### DISCUSSION TITLE
v2/zh/guide/interface/icon

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#494] [Announcements]
# v2/zh/guide/interface/icon

整个主题在多处都添加了 FontClass / 图片 格式图标的支持。 目前你可以使用 iconfont、iconify 和 fontawesome 为你的项目添加图标，可以设置自己的图标资源。 同时，png/svg 格式的图标也是支持的。你可以使用完整 URL 或路径名来添加图标。 我们推荐使用 iconify 或 fontawesome。

https://vuepress-theme-hope.gitee.io/v2/zh/guide/interface/icon.html

**Comment by S-sherry-R** [comment]
我使用自己生成的iconfont资源，图标active的时候不变颜色，怎么回事？

**Reply by S-sherry-R** [reply]
我知道为什么了，项目设置->字体格式，取消彩色勾选
